independent-publisher-connectors/OpenNEM/settings.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
@paulculmsee Please remove `"settings.json"` file from the PR which is not required to open-source your connector.